src/cms/fixtures/test_data.json

```diff
@@ -81,7 +81,8 @@
     "model": "cms.language",
     "pk": 1,
     "fields": {
-      "code": "de-de",
+      "slug": "de",
```

Comment:
I hardcoded these values in the API tests, so I think you need to update them there as well:
https://github.com/Integreat/integreat-cms/blob/92027e4a1d57aeb0cb06679a04e90acc0a764f9f/src/cms/tests/views/api_view_test.py#L44

Reply:
Thanks, will do that.